MeetUpApp.Data/AppDataContext.cs

```diff
@@ -15,121 +16,12 @@ public AppDataContext(DbContextOptions<AppDataContext> options) : base(options)
-                .Property(m => m.Name)
-                .IsUnicode()
-                .IsRequired();
-
-            // description column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.Description)
-                .IsUnicode();
-
-            // speaker column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.Speaker)
-                .IsUnicode()
-                .IsRequired();
-
-            // datetime column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.DateTime)
-                .IsRequired();
-
-            // country column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.Сountry)
-                .IsUnicode()
-                .IsRequired();
-
-            // state column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.State)
-                .IsUnicode()
-                .IsRequired(false);
-
-            // city column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.City)
-                .IsUnicode()
-                .IsRequired();
-
-            // street column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.Street)
-                .IsUnicode()
-                .IsRequired();
-
-            // house column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.House)
-                .IsUnicode()
-                .HasMaxLength(20)
-                .IsRequired();
-
-            // postcode column
-            modelBuilder.Entity<Meetup>()
-                .Property(m => m.PostCode)
-                .HasMaxLength(6)
-                .IsFixedLength()
-                .IsRequired(false);
-        }
-
-        private static void BuildUserTable(ModelBuilder modelBuilder)
-        {
-            // id column
-            modelBuilder.Entity<User>()
-                .HasKey(u => u.Id)
-                .IsClustered();
-
-            modelBuilder.Entity<User>()
-                .Property(u => u.Id)
-                .UseIdentityColumn()
-                .IsRequired();
-
-            // name column
-            modelBuilder.Entity<User>()
-                .HasIndex(u => u.Name)
-                .IsUnique();
-
-            modelBuilder.Entity<User>()
-                .Property(u => u.Name)
-                .HasMaxLength(20)
-                .IsUnicode()
-                .IsRequired();
-
-            // passwordhash column
-            modelBuilder.Entity<User>()
-                .Property(u => u.PasswordHash)
-                .HasMaxLength(88)
-                .IsFixedLength()
-                .IsRequired();
-
-            // salt column
-            modelBuilder.Entity<User>()
-                .Property(u => u.Salt)
-                .HasMaxLength(44)
-                .IsFixedLength()
-                .IsRequired();
+            new UserEntityTypeConfiguration()
+                .Configure(modelBuilder.Entity<User>());
```

Comment:
please use `modelBuilder.ApplyConfigurationsFromAssembly` method to register all entity configurations at once